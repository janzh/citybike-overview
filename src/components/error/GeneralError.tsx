import React from "react";
import styled from "styled-components";

const texts = {
    error: "Beklager det har skjedd en feil, prÃ¸v igjen senere.",
};

const Container = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
`;

const GeneralError = () => (
    <Container>
        <p>{texts.error}</p>
    </Container>
);

export default GeneralError;
