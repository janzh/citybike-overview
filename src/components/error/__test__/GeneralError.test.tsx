import { render } from "@testing-library/react";
import React from "react";

import GeneralError from "../GeneralError";

describe("GeneralError", () => {
    it("should render without failure", () => {
        const { asFragment } = render(<GeneralError />);
        expect(asFragment()).toMatchSnapshot();
    });
});
