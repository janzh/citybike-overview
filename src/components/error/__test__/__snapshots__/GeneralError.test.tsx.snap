// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeneralError should render without failure 1`] = `
<DocumentFragment>
  <div
    class="sc-beyTiQ MeDgu"
  >
    <p>
      Beklager det har skjedd en feil, pr√∏v igjen senere.
    </p>
  </div>
</DocumentFragment>
`;
